format/FlightSql.proto

```diff
@@ -1778,6 +1794,47 @@ message CommandPreparedStatementUpdate {
   bytes prepared_statement_handle = 1;
 }
 
+/*
+ * Represents a bulk ingestion request. Used in the command member of FlightDescriptor
+ * for the the RPC call DoPut to cause the server load the contents of the stream's
+ * FlightData into the target destination.
+ */
+message CommandStatementIngest {
+  option (experimental) = true;
+
+  // Describes the behavior for loading bulk data.
+  enum IngestMode {
+    // Ingestion behavior unspecified.
+    INGEST_MODE_UNSPECIFIED = 0;
```

Comment:
Why do we want to support this?

Reply:
I think there may be some combinations of other options for which it may not make sense for the client to have to specify an IngestMode explicitly. For example some server implementations of `temporary` may abstract this detail away, as may be the case with certain backend-specific `options`. This was originally proposed in the [adbc issue](https://github.com/apache/arrow-adbc/issues/1107). Does this align with the use-cases you had in mind @lidavidm?

Reply:
This is just Protobuf convention. The server should error.